src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx

```diff
@@ -179,12 +184,25 @@ export default class MarketOutcomeDepth extends Component {
           // const y = mouse[1]
 
           // Determine closest order
-          const hoveredPrice = this.state.yScale.invert(mouse[1])
+          const hoveredPrice = this.state.yScale.invert(mouse[1]).toFixed(this.props.fixedPrecision)
 
           this.props.updateHoveredPrice(hoveredPrice)
         })
         .on('click', () => {
-          console.log('TODO -- fill order form')
+          const mouse = d3.mouse(d3.select('#outcome_depth').node())
```

Comment:
Is this "scoped" to the current component somehow?

Reply:
Which aspect are you referencing exactly?

_Guessing_ you’re referencing the `select`?  

If so, that is querying the DOM globally; thus, the `#`; however, it only matches the first result.

Reply:
Might make sense to make that ID more specific?

Reply:
I meant the selector. Will we ever have 2 of these up at once? If so, will that cause a problem?

Reply:
Re: 2 at the same time, no for the depth chart (at least per the mocks)

Reply:
Cool.